### PR TITLE
Generic seedlot upload

### PR DESCRIPTION
**Pull request by titima15**
Description 
-----------------------------------------------------

Checklist 
---------------------------------------------------------------------------------------
- [ ] Refactoring only
- [ ] Documentation only
- [ ] Fixture update only
- [ ] Bug fix
  - [ ] The relevant issue has been closed.
  - [ ] Further work is required.
- [ ] New feature
  - [ ] Relevant tests have been created and run.
  - [ ] Data was added to the fixture
    - [ ] Data was added via a patch in `/t/data/fixture/patches/`.
  - [ ] User-Facing Change
    - [ ] The user manual in `/docs` has been updated.
  - [ ] Any new Perl has been documented using **perldoc**.
  - [ ] Any new JavaScript has been documented using **JSDoc**.
  - [ ] Any new _legacy_ JavaScript has been moved from `/js` to `/js/source/legacy`.

### CODE REVIEW

**Comment by titima15** [comment]
@dwaring87 I don't see the comment that you mentioned in the lab meeting.

**Review by dwaring87 — CHANGES_REQUESTED** [review]

**Comment by dwaring87** [comment]
@titima15 How about now?  I guess I need to submit the review before you can see the comments...

File: lib/CXGN/Stock/Seedlot/ParseUpload/Plugin/SeedlotFromAccessionGeneric.pm [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+        my $accession_name = $row->{'accession_name'};
+        my $amount = $row->{'amount'};
+        my $weight = $row->{'weight_gram'};
+        my $seedlot_source = $row->{'source'};
+
+        if ($seedlot_name =~ /\s/ || $seedlot_name =~ /\// || $seedlot_name =~ /\\/ ) {
+            push @error_messages, "Cell A$row_num: seedlot_name must not contain spaces or slashes.";
+        }
+
+        if ($duplicated_seedlot_names{$seedlot_name}) {
```
[the remark this answers is not in the document]
Comment (dwaring87) [reply lib/CXGN/Stock/Seedlot/ParseUpload/Plugin/SeedlotFromAccessionGeneric.pm:68]: The same applies to the other plugin

[the remark this answers is not in the document]
Comment (titima15) [reply lib/CXGN/Stock/Seedlot/ParseUpload/Plugin/SeedlotFromAccessionGeneric.pm:68]: oops, I forgot. Thanks. I already fixed.

**Review by dwaring87 — APPROVED** [review]
Thanks!

**Review by lukasmueller — APPROVED** [review]
:-)

**Pull request merged**